src/plugins/objmon/objmon.cpp

```diff
@@ -150,27 +150,20 @@ static event_response_t cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
 {
 
     objmon* o = (objmon*)info->trap->data;
-    vmi_instance_t vmi = drakvuf_lock_and_get_vmi(drakvuf);
     struct ckey ckey = {};
 
-    access_context_t ctx;
-    memset(&ctx, 0, sizeof(access_context_t));
```

Comment:
Why are you removing the memset?

Reply:
Because of the designated initializer do this already.